openebs/openebs-provisioner.go

```diff
@@ -43,8 +43,13 @@ const (
 	// BetaStorageClassAnnotation represents the beta/previous StorageClass annotation.
 	// It's currently still used and will be held for backwards compatibility
 	BetaStorageClassAnnotation = "volume.beta.kubernetes.io/storage-class"
+	//defaultFSType
+	defaultFSType = "ext4"
 )
 
+// validFSType represents the valid fstype supported by openebs volume
+var validFSType = map[string]bool{"ext4": true, "xfs": true}
```

Comment:
Can this valid list of options be passed as parameters to the pod? Extending this further, it will be good if all default options can be passed as configuration parameters to container.

Reply:
Added `OPENEBS_VALID_FSTYPE` env variable which can be exported in provisioner POD 
to add fstype in a actual valid fstype supported list.